source/Cosmos.System2/Graphics/Bitmap.cs

```diff
@@ -55,6 +55,15 @@ public Bitmap(uint Width, uint Height, byte[] pixelData, ColorDepth colorDepth)
             }
         }
         
+        public Bitmap(uint Width, uint Height, int[] pixelData, ColorDepth colorDepth) : base(Width, Height, colorDepth)
```

Comment:
Please add documentation to the function call. I also dont think this function call needs ColorDepth and can always use ColorDepth32